crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -353,6 +353,10 @@ define_tedge_config! {
             /// forwarded to Cumulocity on the `s/us` topic
             #[tedge_config(example = "c8y", default(function = "c8y_topic_prefix"))]
             topic_prefix: TopicPrefix,
+
+            /// The amount of time after which the bridge should send a ping if no other traffic has occured
+            #[tedge_config(example = "60s", default(from_str = "60s"))]
+            keepalive_interval: SecondsOrHumanTime,
```

Comment:
Optional:
```suggestion
            keep_alive_interval: SecondsOrHumanTime,
```

Reply:
I take that back, after noticing that the term `keepalive` is kinda like the norm across technologies like HTTP and GRPC.